module/applications/advancement/advancement-manager.mjs

```diff
@@ -327,7 +370,7 @@ export default class AdvancementManager extends Application {
    * @param {string} classItem      Class being changed.
    * @param {number} levelDelta     Levels by which to increase or decrease the class.
    * @returns {AdvancementManager}  Manager with new steps.
-   * @private
+   * @protected
```

Comment:
Surely a mistake this was annotated `@private` in the first place, since it doesn't have an underscore? Looks like public API to me.